src/parse/tnt.py

```
@@ -1,18 +1,160 @@
+        }
+    )
+    protein_df['description'] = protein_df['description'].apply(
+        lambda x: x[:50] + '...' if len(x) > 50 else x
+    )
+    file_manager.store_data(dataset_id, 'protein_dfs', protein_df)
+
+    # tag_table
+
+    # Process tag df into a linear data format
+    new_tag_df = {c : [] for c in tag_df.columns}
+    for i, row in tag_df.iterrows():
```

Ruff (0.8.2)

Loop control variable `i` not used within loop body

(B007)